hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -107,6 +107,58 @@
       '''
     }
   ],
+  features: [
+    { name: "SPIHOST.USECASE.SERIALNORFLASH",
+      desc: '''
+        The SPI host block should be able to talk to serial NOR flash devices.
+      '''
+    },
+    { name: "SPIHOST.USECASE.PASSTHROUGH",
+      desc: '''
+        The SPI host block should be able to talk to serial NOR flash devices.
+      '''
+    },
+    { name: "SPIHOST.RATE.STANDARD",
+      desc: '''
+        Standard SPI data rate.
+      '''
+    },
+    { name: "SPIHOST.RATE.DUAL",
+      desc: '''
+        Dual SPI data rate.
+      '''
+    },
+    { name: "SPIHOST.RATE.QUAD",
+      desc: '''
+        Quad SPI data rate.
+      '''
+    },
+    { name: "SPIHOST.CONFIG.CPOL",
+      desc: '''
+        Polarity of the SPI.
+      '''
+    },
+    { name: "SPIHOST.CONFIG.CLOCKDIV",
+      desc: '''
+        Slows down the clock of the SPI.
+      '''
+    },
+    { name: "SPIHOST.EVENT.WATERMARK",
+      desc: '''
+        Raise an event on certain watermarks.
```

Comment:
Another nitty comment, but I think this should probably be something like "The block can be configured to raise an event on certain watermarks."?

Similarly for the other two SPIHOST.EVENT.* items.

Reply:
Thanks, hopefully I have resolved all of these now.